### DISCUSSION TITLE
Suggestion: show RDS deviation level, and total deviation.

### DISCUSSION THREAD

**Discussion by AaronJowitt** [PE5PVB/TEF6686_ESP32#276] [Ideas]
If available, useful to have an option to show total FM carrier deviation value, with 1s peak hold. i.e. 75 kHz.  RDS (and pilot) deviation may also be useful, given that RDS injection levels vary greatly among broadcasters, impacting RDS decoding.

**Comment by andimik** [comment]
Please note, that you would like to have SDR functions on the TEF.

**Comment by PE5PVB** [comment]
This is not possible. You should have MPX audio directly to the ESP32 for FTT processing and I2S to a DA converter to get audio. Current hardware designs don't have this.
